src/Microsoft.Azure.WebJobs.ServiceBus/Bindings/ServiceBusBinding.cs

```diff
@@ -48,7 +51,8 @@ public bool FromAttribute
             {
                 Account = _account,
                 MessageSender = messageSender,
-                AccessRights = _accessRights
+                AccessRights = _accessRights,
```

Comment:
very tempted to just throw the actual attribute on ServiceBusEntity and pull AccessRights & isQueue from that.  Also fine with changing isQueue to an enum of EntityTypes or something